torchgeo/datasets/forestdamage.py

```diff
@@ -0,0 +1,327 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Forest Damage dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, Tuple
+from xml.etree import ElementTree
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, download_and_extract_archive, extract_archive
+
+
+def parse_pascal_voc(path: str) -> Dict[str, Any]:
```

Comment:
This seems like something to put in `torchgeo.datasets.utils`

Reply:
In the FAIR1M dataset, annotations also come in PASCAL VOC format, but the xml files differ so, like in the FAIR1M dataset, I put the parsing function in `forestdamage.py` itself.